demo-transfer-dashboard/src/Components/Form.jsx

```diff
@@ -157,12 +161,11 @@ const Form = () => {
             {transfers.map((transfer, index) => (
               <tr key={index}>
                 <td>{transfer.patient_id}</td>
-                <td>{transfer.originPT}</td>
-                <td>{transfer.receivingPT}</td>
+                <td>{transfer.transfer_to}</td>
                 <td
                   className={`status ${transfer.status === 'Transferred' ? 'transferred' : 'failed'}`}
```

Comment:
`status` comes from BullMQ, can be one of `'completed' | 'failed' | 'active' | 'delayed' | 'prioritized' | 'waiting' | 'waiting-children'`. I suggest the logic here have classes for `completed`, `failed`, and as a default `ongoing`.